PrebidMobile/ConfigurationAndTargeting/UserConsentDataManager.swift

```diff
@@ -51,16 +55,26 @@ public class UserConsentDataManager: NSObject {
      Integer flag indicating if this request is subject to the COPPA regulations
      established by the USA FTC, where 0 = no, 1 = yes
      */
-    public var subjectToCOPPA: Bool? {
+    var subjectToCOPPA: Bool? {
         set { UserDefaults.standard.set(newValue, forKey: PB_COPPAKey) }
         get { UserDefaults.standard.object(forKey: PB_COPPAKey) != nil ? UserDefaults.standard.bool(forKey: PB_COPPAKey) : nil }
```

Comment:
need to remove writing this value into UserDefaults. Use the same approach as for the TCF signal here as well.